### ISSUE TITLE
Move the reposity to a better long-term origin

### ISSUE THREAD

**Issue by MansMeg** [welfare-state-analytics/riksdagen-corpus#238]
We should store the corpus on a better repository long term. KB has said that they are not interested. Maybe setup a solution with riksdagen library instead.

**Comment by ninpnin** [comment]
We can have multiple remotes.

**Comment by MansMeg** [comment]
Yes. But we need at least one long-term remote (long-term = financibg after swerik ends).
